src/EnergyPlus/CMakeLists.txt

```diff
@@ -602,6 +602,8 @@ set(SRC
     TARCOGParams.hh
     TarcogShading.cc
     TarcogShading.hh
+    Timer.cc
+    Timer.hh
```

Comment:
Hey I've seen that before!  

https://github.com/NREL/EnergyPlusArchive/blob/master/SourceCode/Timer.h